website_backend_views/model/ir_qweb.py

```diff
@@ -24,7 +24,7 @@
 
 
 class IrQweb(models.Model):
```

Comment:
@hbrunn I prefer to quote Einstein about simplicity. Though his sentence has less soul.